### PR TITLE
Improve PR handling by adding a policy bot

### PR DESCRIPTION
**Pull request by nhhagen**

### CODE REVIEW

File: .policy.yml
Comment (nhhagen) [inline .policy.yml:32]: We might be able to make this into a shared rule: https://github.com/palantir/policy-bot#configuration

> The file is read from the most recent commit on the target branch of each pull request.
>
> The file may contain a reference to a policy in a different repository (see [Remote Policy Configuration](https://github.com/palantir/policy-bot#remote-policy-configuration).)
>
> If the file does not exist in the repository, policy-bot tries to load a shared policy.yml file at the root of the .github repository in the same organization. You can change this path and repository name when running your own instance of the server.
>
>If a policy does not exist in the repository or in the shared organization repository, policy-bot does not post a status check on the pull request. This means it is safe to enable policy-bot on all repositories in an organization.

But lets get something that works first

File: .policy.yml
Comment (nhhagen) [inline .policy.yml:17]: It would be nice to set this to random users, but how do we get the policy bot to request more than 1 review?

**Comment by nhhagen** [comment]
Adding policy-bot will not automatically merge Dependabot PRs that are approved. Unless we add something like https://github.com/palantir/bulldozer users will still have to approve with the comment:

```markdown
@dependabot squash and merge
```

For internal repos we have automation that can trigger on approval or policy-bot status check, but it would be nice if we could run the same setup for public and internal repos.

File: .policy.yml
Comment (AtzeDeVries) [reply .policy.yml:32]: yeah shared rules can be very useful.

**Comment by AtzeDeVries** [comment]
yeah including bulldozer and maybe also stargazer would be very useful. You can label PRs using stargazer and merged based on labes.

File: .policy.yml
Comment (AtzeDeVries) [reply .policy.yml:17]: ah you want to request more people? ah that i've been wondering as well.

Comment (AtzeDeVries) [reply .policy.yml:17]: looks the amount of users is set the count in the rule to https://github.com/palantir/policy-bot/blob/909e077c5058753b78a2f6dc8b1965488f30fac0/policy/reviewer/reviewer.go#L272

Comment (AtzeDeVries) [reply .policy.yml:17]: we could make a feature request to add a percentage (or number) to the setting of random-users. Feedback might be that we need to use teams

Comment (nhhagen) [reply .policy.yml:17]: Should we set it to teams and see how it behaves?

Comment (AtzeDeVries) [reply .policy.yml:17]: that will be similar do how it works with CODEOWNERS. So also very simular to 'all-users'. The 'all-users' is actually very similar to 'teams' but does not require the team to have push access to the repo (which all org users already have)

File: .policy.yml
Comment (nhhagen) [reply .policy.yml:32]: Shall we do that now, or promote later when we have a bit more experience?

File: .policy.yml
Comment (nhhagen) [reply .policy.yml:17]: aha, what do we prefer in that case, that the reviewers have concrete push access or not for a repo like `go-datadog-lib`?

File: .policy.yml
Comment (AtzeDeVries) [reply .policy.yml:32]: no later i would guess

File: .policy.yml
Comment (AtzeDeVries) [reply .policy.yml:17]: for the future i would like to not use the `teams` feature since that requires extra (and unuseful for us) config on the repository. So i prefer `all-useres` or `random`

**Review by AtzeDeVries — DISMISSED** [review]

File: .policy.yml
Comment (misund) [reply .policy.yml:32]: It makes sense to keep it local for a short while, but I hope the experience is good enough to escalate to org level quickly

File: .policy.yml
Comment (nhhagen) [reply .policy.yml:17]: `all-users` it is.

File: .policy.yml
Comment (nhhagen) [reply .policy.yml:32]: then we keep it like this for now

**Review by AtzeDeVries — APPROVED** [review]

**Review by misund — APPROVED** [review]

**Pull request merged**
